qtpy/enums_compat.py

```diff
@@ -37,3 +37,24 @@ def promote_enums(module):
                     continue
                 for name, value in attrib.__members__.items():
                     setattr(klass, name, value)
+
+if PYQT5:
+    def demote_enums(module):
+        """
+        Search unscoped enums in the given module and allow scoped access.
+
+        Special for PyQt5==5.9.*
+        """
+        class LookUp:
+            def __init__(self, lookup_location):
+                self._ll = lookup_location
+
+            def __getattr__(self, what):
+                return getattr(self._ll, what)
```

Comment:
```suggestion
                self._lookup_location = lookup_location

            def __getattr__(self, what):
                return getattr(self._lookup_location, what)
```

Any reason not to use a more descriptive name here?

Reply:
The “l”s stand for my laziness. That's the only reason.

Actually, I wrote the code on Windows XP for `PyQt5==5.6` (that was painful). There was next to no IDE, so I used as short variable names as were possible, keeping the code understandable.

Reply:
Windows XP :scream:

Oh wow, how long ago did you write this code?

Reply:
Some 2.5 _months_ ago. I had to port lots of code from Python 3.11 with PySide6 back to Python 3.4 and PyQt5 5.6. For that, I had to make up a tool to speed it up. The tool worked so well that I didn't have to fix anything manually most of the time. That was so great and useless project!

Reply:
Wow, I can't imagine what sort of project would require that...considering it's been EoL for nearly a decade now (aside from the last embedded/POS editions, which "only" went EOL 4 years ago). In any case, doesn't sound like fun, though seems like your tool managed to make it less of a pain.